conda/recipes/ucxx/meta.yaml

```
@@ -163,6 +177,12 @@ outputs:
         - {{ pin_subpackage('libucxx', exact=True) }}
         - ucx
         - rmm =23.08
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - cuda-cudart-dev
+        {% endif %}
+        - cuda-version {{ cuda_spec }}
```

The `cuda-version` in `host` should be a specific value corresponding to the CI matrix entry. If you want to support a wide range of CUDA minor versions at runtime, add `- {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}` in the run section -- assuming that CUDA is a hard dependency of `ucxx` (we discussed above that it isn't a hard dependency of `libucxx`).
```suggestion
        - cuda-version {{ cuda_version }}
```

---

It isn't a hard-dependency, but we still want to support `>=11.2,<12.0`. Is `pin_compatible` not necessary then?

---

This needs to match the CI job's CUDA version:
```suggestion
        - cuda-version {{ cuda_version }}
```